.github/workflows/schema-spy.yml

```diff
@@ -0,0 +1,38 @@
+name: Schema Spy report generation
+
+on:
+  schedule:
+    - cron: "0 7 * * TUE" # Every tuesday at 07:00 UTC
+
+  workflow_dispatch:
+
+jobs:
+  schema-spy:
+    name: Schema Spy report generation
+    runs-on: moj-cloud-platform
+    timeout-minutes: 240
+    steps:
+      - uses: actions/checkout@v3
+      - name: run schema spy
+        shell: bash
+        env:
+          DB: ${{ secrets.SCHEMA_SPY_DB }}
+          HOST: ${{ secrets.SCHEMA_SPY_HOST }}
+          PORT: ${{ secrets.SCHEMA_SPY_PORT }}
+          SCHEMA: ${{ secrets.SCHEMA_SPY_SCHEMA }}
+          USERNAME: ${{ secrets.SCHEMA_SPY_USERNAME }}
+          PASSWORD: ${{ secrets.SCHEMA_SPY_PASSWORD }}
+        run: run.sh
+        working-directory: schema-spy
+
+      - name: Publish HTML report
+        uses: JamesIves/github-pages-deploy-action@v4
+        if: always()
```

Comment:
Don't think we want this line?  If the SchemaSpy generation fails, should we still try to publish the report?